tensorflow-core/tensorflow-core-api/src/main/java/org/tensorflow/ConcreteFunction.java

```diff
@@ -39,16 +57,87 @@
  */
 public class ConcreteFunction implements AutoCloseable {
 
+  private static TF_Operation outputHandle(Operand<?> operand) {
```

Comment:
Nit: can we move the private parts at the bottom of the class? It's the Google way to do it and I kind of like it now :)